src/components/pages/JudgePage/JudgePage.js

```diff
@@ -287,7 +295,7 @@ export default function JudgePage(props) {
             </h1>
             {judge.birthdate ? <p>Birthdate: {judge.birthdate}</p> : null}
 
-            <p>County: {judge.judge_county}</p>
+            <p>County: {judge.county}</p>
```

Comment:
This was a simple error fix that was unrelated to the main focus of this update. The county data was not rendering because "judge_county" did not correspond to any column in our "judges" table. I corrected it to the proper column name.